Sources/ProjectSpec/Scheme.swift

```diff
@@ -155,6 +159,7 @@ extension Scheme.Test: JSONObjectConvertible {
         config = try jsonDictionary.json(atKeyPath: "config")
         gatherCoverageData = jsonDictionary.json(atKeyPath: "gatherCoverageData") ?? false
         commandLineArguments = jsonDictionary.json(atKeyPath: "commandLineArguments") ?? [:]
+        testables = jsonDictionary.json(atKeyPath: "testables") ?? []
```

Comment:
The name `testables` comes from the scheme xml which makes sense. Other name candidates include `testTargets`, or `tests`.
My personal preference would be `tests`, as it's clear and matches the Xcode UI. Thoughts?

Note that in the future

Reply:
well, I'm fine with both `testTargets` and `tests`, although when I look to the project spec it look a bit repetetive:

```yaml
STCore:
  build:
    targets:
      - target: STCore
        buildTypes: all
  test:
    tests: [STCoreTests]
    config: Staging-Debug
```

that might be a bit less repetitive:
```yaml
STCore:
  build:
    targets:
      - target: STCore
        buildTypes: all
  test:
    targets: [STCoreTests]
    config: Staging-Debug
```

Reply:
Yeah, I like targets! 👍

Reply:
So rename `Test.testables` to `targets` (and the yaml property)